frontend/src/utils/api/index.js

```diff
@@ -21,11 +21,6 @@ async function responseCheck(res) {
 function request (path, { headers, method, data }) {
   // eventually add cors and auth headers
   const url = `${API_URL}${path}`
-
-  if (method === 'GET') {
```

Comment:
I still don't think sending the auth token with every req is good.

Reply:
I agree, but for now I think its okay